styles.css

```diff
@@ -439,3 +439,35 @@ body.dark .footer::before {
         margin-bottom: 10%;
     }
 }
+
+/* --------------------------------------------------------ADDED A TRANSFORM EFFECT--------------------------------------------- */
+body.light .effect{
+    transition: text-decoration 0.2s ;
+}
+body.light .effect:hover{
+    text-decoration: underline;
+    color:#D77B3D;
+}
+
+body.light #effect{
+    transition: text-decoration 0.2s ;
+}
+body.light #effect:hover{
+    text-decoration: underline;
+    color:#D77B3D;
+}
+body.dark .effect{
+    transition: text-decoration 0.2s ;
+}
+body.dark .effect:hover{
+    text-decoration: underline;
+    color: #f1e0c6;
+}
+
+body.dark #effect{
+    transition: text-decoration 0.2s ;
+}
+body.dark #effect:hover{
+    text-decoration: underline;
```

Comment:
I have looked into the hosted site and it seems like the transition is happening instantaneously (or it's not working properly). You also might want to use `all` as transition property, as the transition in color might also look good.